test/html/checkCookieExists.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>checkCookieExist- Test</title>
+</head>
+
+<body>
+    <h1>checkCookieExist</h1>
+
+    <p id='existing-cookie'></p>
+
+    <p id='holding-no-cookies'></p>
+    
+    <script>
+
+        //set a cookie with a value
+        setCookie("existing-cookie", "Cookie with a value");
+        //set a second cookie with the same value
+        setCookie("second-existing-cookie", "Cookie with a value");
+        //modifies existing cookie value
+        setCookie("second-existing-cookie", "Cookie with a new value");
+        //set a cookie with no value
+        setCookie("undifined-value-cookie", );
+        
+        displayCookie("not-existing-cookie");
+
+        function setCookie(name, value) {
+            document.cookie = name + "=" + value;
+            var x = document.cookie
```

Comment:
This can be a `const` since its value isn't set again after it's declared:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

```suggestion
            const x = document.cookie
```